units/getty@.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Getty on %I
+DefaultDependencies=no
+
+[Service]
+Environment=TERM=xterm-256color
+EnvironmentFile=/etc/getty/username.env
+ExecStart=-/sbin/agetty -o '-- ${USERNAME}' --noclear --skip-login - ${TERM}
```

Comment:
Does using `${USERNAME}` rather than `\u` mean that no matter what username is entered, we always send the user-data-supplied username to `login`? That's arguably a minor loss to security, since they wouldn't have to guess the valid username.

Reply:
We can potentially also keep the `-p` flag if we use an `UnsetEnvironment=<blah>` directive below to remove problematic environment variables.

That would allow us to pass through variables like `NO_PROXY` that are set in the ssh case, which I'd guess might be discarded otherwise.

Reply:
I tried `UnsetEnvironment` and it didn't unset `HOME=/root`.

Interestingly enough, without `-p` the proxy env vars still show up fine.

```
[erikson@admin]$ env
HOSTNAME=
TERM=xterm-256color
SHELL=/bin/bash
HISTSIZE=1000
NO_PROXY=localhost,127.0.0.1,191bc2a1c5d1e9e863c973a7bdfb88d0.yl4.us-west-2.eks.amazonaws.com,.cluster.local
USER=erikson
...
MAIL=/var/spool/mail/erikson
PATH=/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/home/erikson/.local/bin:/home/erikson/bin
PWD=/home/erikson
HISTCONTROL=ignoredups
SHLVL=1
HOME=/home/erikson
no_proxy=localhost,127.0.0.1,191bc2a1c5d1e9e863c973a7bdfb88d0.yl4.us-west-2.eks.amazonaws.com,.cluster.local
LOGNAME=erikson
_=/usr/bin/env

```